### PR TITLE
Support building against LLVM 3.9

### PR DESCRIPTION
**Pull request by jeroenk**

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -2,8 +2,8 @@ Requirements:
 -------------
 
   * A C++ compiler (both gcc and clang have been tested).
-  * LLVM (versions 2.9 and 3.4 have been tested, but versions 3.0, 3.1,
-          3.2, 3.3, 3.5, 3.6, 3.7, and 3.8 should also work).
+  * LLVM (versions 2.9 and 3.4 have been tested, but versions all versions
```
Comment (s-falke) [inline INSTALL]: Could you please drop the first occurrence of "version"?  Thanks!

Comment (s-falke) [inline INSTALL]: D'Oh, I meant the second occurrence of "versions"...

**Comment by jeroenk** [comment]
Should be fixed now

**Comment by s-falke** [comment]
Thanks!

**Pull request merged**